packages/web-config-server/src/apiV1/report.js

```diff
@@ -67,8 +72,18 @@ export class ReportHandler extends DataAggregatingRouteHandler {
       requestQuery.endDate = this.endDate;
     }
 
-    const { results } = await reportConnection.fetchReport(reportCode, requestQuery);
-    return Array.isArray(results) ? { data: results } : { ...results };
+    const { results, ...metadata } = await reportConnection.fetchReport(reportCode, requestQuery);
+    const { period } = metadata;
+    const reportMetadata = {};
+    if (period?.reportStart) {
+      reportMetadata.dataStartDate = periodToDateString(period.reportStart, false);
+    }
+    if (period?.reportEnd) {
+      reportMetadata.dataEndDate = periodToDateString(period.reportEnd, true);
+    }
+    return Array.isArray(results)
+      ? { data: results, ...reportMetadata }
+      : { ...results, ...reportMetadata };
```

Comment:
As of this change, we now receive the following when requesting data for a dashboard:
```
{
  results, // data for report
  startDate, // requested start date
  endDate, // requested end date
  dataStartDate, // date of earliest data from report
  dataEndDate, // date of latest data from report
}
```